src/libs/actions/Wallet.js

```diff
@@ -321,6 +321,36 @@ function answerQuestionsForWallet(answers, idNumber) {
     );
 }
 
+function requestPhysicalExpensifyCard(cardID, {legalFirstName, legalLastName, phoneNumber, address: {city, street, zip}}) {
+    const params = {
+        legalFirstName,
+        legalLastName,
+        phoneNumber,
+        addressCity: city,
+        addressStreet: street,
+        addressZip: zip,
+    };
+    const onyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.CARD_LIST,
+                value: {
+                    [cardID]: {
+                        state: 4, // NOT_ACTIVATED
+                    },
+                },
+            },
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.PRIVATE_PERSONAL_DETAILS,
+                value: params,
```

Comment:
I am not sure if these params can be dropped here as is...

Especially the `address` you can see here it has this format:

https://github.com/Expensify/App/blob/9e35d91dd59b9233a584150a9a19f1ef458e9622/src/types/onyx/PrivatePersonalDetails.ts#L1-L16

So what we are storing might be different than what we send to the API.

Reply:
Besides that, you're right, I need to send the data directly without mapping it

Reply:
@shawnborton The designs for the confirmation page don't have a fixed size applied to the items of the list, as you can see below:
![Screenshot 2023-10-18 at 12 02 32](https://github.com/Expensify/App/assets/48553277/089e4d46-761c-4dbd-8397-30623f93c03c)

Should these items size match the existing ones on the app? And if so, could you update the designs, please?

Reply:
@pac-guerreiro those should just be existing components in the design system called "push-to-page" rows. They might be called something like MenuOption in the code? cc @grgia for help there

Reply:
But you should be able to reuse existing code for these and not need to worry about building something new.

Reply:
@shawnborton thanks for the response! I'm already reusing the right component for this but I was applying the styles from Figma, so I removed these styles and tweaked the whole layout to work correctly 😄 

This is the current state:

![Screenshot 2023-10-18 at 12 56 26](https://github.com/Expensify/App/assets/48553277/14563216-eee4-4a8c-8841-a9369ce569a3)

Reply:
Sweet, that feels pretty good to me!